cg-spec/editors_draft.html

```diff
@@ -2229,38 +2238,33 @@ <h2>Merging RDF-star graphs</h2>
       <h2>Remarks</h2>
 
       <section>
-        <h2>Combining RDF-star graphs</h2>
+        <h2>Implementing the RDF-star semantics</h2>
 
-        <p>Care must be taken when <a>RDF graphs</a> that result from <a>RDF-star graphs</a> are combined through union or <a>merging</a>. Given two <a>RDF-star graphs</a> |G| and |H|, it may be the case that |unstar|(|G| ∪ |H|) ≠ |unstar|(|G|) ∪ |unstar|(|H|). More precisely, if |G| and |H| contain the same <a>embedded triple</a>, this triple will be mapped to a single blank node in |unstar|(|G| ∪ |H|), but in two potentially different blank nodes in |unstar|(|G|) ∪ |unstar|(|H|). These blank nodes will need to be unified in order to get the correct entailments.</p>
+        <p>It is important to notice that the definitions given <a href="#entailment-of-rdf-star-graphs">above</a> for <a>satisfiability</a> and <a>entailment</a> of <a>RDF-star graphs</a>, do not mandate implementations to actually transform <a>RDF-star graphs</a> using the |unstar| mapping. It should be possible to implement the semantics by handling directly RDF-star's <a href="#concepts">abstract syntax</a>.</p>
 
       </section>
+
       <section>
-        <h2>Considerations on interoperability</h2>
+        <h2>Combining RDF-star graphs</h2>
 
-        <p>The special properties (|ST|, |PT|, etc.), used in the mapping for representing <a>embedded triples</a> in plain RDF, are deliberately not specified. As a consequence, although any <a>RDF-star graph</a> |G| is semantically equivalent to an <a>RDF graph</a> |unstar|(|G|), that latter graph is implementation dependent, as different systems will use a different concrete <a>IRI</a> for each special property.</p>
+        <p>For implementations that rely on the |unstar| mapping defined <a href="#mapping">above</a>, care must be taken when the resulting <a>RDF graphs</a> are combined through union or <a>merging</a>. Given two <a>RDF-star graphs</a> |G| and |H|, it may be the case that |unstar|(|G| ∪ |H|) ≠ |unstar|(|G|) ∪ |unstar|(|H|).</p>
```

Comment:
```suggestion
        <p>For implementations that rely on the |unstar| mapping defined <a href="#mapping">above</a>, care must be taken when the resulting <a>RDF graphs</a> are combined through union or <a>merge</a>. Given two <a>RDF-star graphs</a> |G| and |H|, it may be the case that |unstar|(|G| ∪ |H|) ≠ |unstar|(|G|) ∪ |unstar|(|H|).</p>
```